osquery/filesystem/filesystem.cpp

```diff
@@ -131,10 +131,9 @@ Status readFile(const fs::path& path,
   auto read_max = static_cast<off_t>(FLAGS_read_max);
   if (file_size > read_max) {
     if (!dry_run) {
-      LOG(WARNING) << "Cannot read file that exceeds size limit: "
-                   << path.string();
-      VLOG(1) << "Cannot read " << path.string()
-              << " size exceeds limit: " << file_size << " > " << read_max;
+      return Status::failure(
+          "Cannot read " + path.string() + " size exceeds limit: " +
+          std::to_string(file_size) + " > " + std::to_string(read_max));
```

Comment:
The issue here though is that it's not guaranteed that all the readFile callers will log this error, so we should ensure that they are, to not change the end result.

Reply:
ok, i will look through all the call sites and see if it's logged properly.

Reply:
ok, i've set up an extra `log` var to control whether readFile (and friends) log directly or log via status using a local logger.  I've then wired this up to not log by default in all the tables I've added.  This was easier than fixing all of the calls of readFile which ignore the returned status.